### PR TITLE
Add IDE (Eclipse) instructions

### PR DESCRIPTION
**Pull request by ojuschugh1**
Add instructions for IDE #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
+Install Maven:
```
Comment (m-reza-rahman) [inline README.md:30]: This should not be needed. Eclipse comes with it's own Maven.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
...
+Before you can use Maven in Eclipse, you need to install it on your system. You can download Maven from the official website (https://maven.apache.org/download.cgi) and follow the installation instructions.
+
+Install the Maven Integration for Eclipse:
```
Comment (m-reza-rahman) [inline README.md:33]: This should not be needed. For Jakarta EE projects, we should specify the Enterprise version of Eclipse.

File: README.md [hunk 3]
```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
...
+The Maven Integration for Eclipse is a plugin that provides integration between Eclipse and Maven. To install the plugin, open Eclipse and go to Help > Eclipse Marketplace. Search for "Maven Integration for Eclipse" and click on the "Install" button. Follow the installation wizard and restart Eclipse when prompted.
+
+Create a new Maven project:
```
Comment (m-reza-rahman) [inline README.md:36]: These are generic instructions. Please provide instructions specific to our archetype.

File: README.md [hunk 4]
```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
...
+To use an archetype in Eclipse, you need to create a new Maven project and select the archetype you want to use. To do this, go to File > New > Project > Maven > Maven Project. In the "New Maven Project" dialog, select "Create a simple project" and click "Next". Enter the Group Id, Artifact Id, and Version for your project, and then click "Next". In the "Select an Archetype" dialog, choose the archetype you want to use and click "Next". You can browse the available archetypes by clicking the "Add Archetype" button and entering the details of the archetype you want to use. Finally, enter any additional information required by the archetype and click "Finish" to create the project.
+
+Configure the Maven project:
```
Comment (m-reza-rahman) [inline README.md:39]: This should not be needed. Eclipse will auto-detect this.

File: README.md [hunk 5]
```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
...
+Once you have created the Maven project, you need to configure it to use the archetype. To do this, right-click on the project in the Project Explorer view and select "Maven > Update Project". In the "Update Maven Project" dialog, select the "Force Update of Snapshots/Releases" checkbox and click "OK". This will download and install the archetype and its dependencies.
+
+Build and run the project:
```
Comment (m-reza-rahman) [inline README.md:42]: This is too generic. The information here should be specific to our archetype.

**Review by m-reza-rahman — CHANGES_REQUESTED** [review]
Please address comments before we can consider merging.

**Comment by ojuschugh1** [comment]
@m-reza-rahman  please review it I have added steps more specific now.
Thanks and Regards

File: README.md [hunk 6]
```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
```
Comment (m-reza-rahman) [inline README.md:29]: This is still basically generic instructions on how to create a Maven project in Eclipse using Archetypes. It really has nothing about our specific project. This needs to outline the specific steps for our project including what the user needs to do in order to move forward with the generated project. At the current trajectory, I estimate days of work before this PR is ready to merge. Please avoid frivolous PR updates and aim for quality as well as completeness. It needlessly consumed everyone's time.

**Comment by ojuschugh1** [comment]
Ok sir will surely be doing more research about it and adding it more in the comment section before the final PR :).
Thanks for giving me your time.It means alot to me :)

**Comment by m-reza-rahman** [comment]
I am trying to do what I can. Please take the time to carefully/completely process feedback provided and take the PR out of draft status when it is really ready for meaningful review.

File: README.md [hunk 7]
```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
...
+
+# If you are using an enterprise version of Eclipse for a Jakarta EE project, you can follow these steps to create a new Maven project with an archetype:
+
+Open Eclipse and select "File > New > Other..." from the menu bar.
```
Comment (m-reza-rahman) [inline README.md:33]: Selecting "Other" is unnecessary. Maven has been a top level choice in Eclipse IDE for some time now.

File: README.md [hunk 8]
```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
+
+# If you are using an enterprise version of Eclipse for a Jakarta EE project, you can follow these steps to create a new Maven project with an archetype:
```
Comment (m-reza-rahman) [inline README.md:31]: This should be explicit about what the setup pre-requisites are. Here is an example: https://github.com/eclipse-ee4j/cargotracker/wiki/Eclipse-IDE.

File: README.md [hunk 9]
```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
...
+
+Open Eclipse and select "File > New > Other..." from the menu bar.
+In the "New" dialog, expand the "Maven" folder and select "Maven Project".
+Click "Next".
+In the "New Maven Project" dialog, select "Create a simple project".
+Click "Next".
+In the "New Maven Project" dialog, enter the Group Id, Artifact Id, and Version for your project.
+Click "Next".
+In the "Select an Archetype" dialog, click the "Add Archetype" button.
```
Comment (m-reza-rahman) [inline README.md:40]: Why is "Add Archetype" needed here? Our archetype will already be in Maven Central or installed as a SNAPSHOT in the local repository.

**Comment by ojuschugh1** [comment]
Thanks for the help!

**Pull request closed without merging**